test/foundry/BlastERC20RebasingYield.t.sol

```diff
@@ -15,27 +14,30 @@ import {MockBlastPoints} from "../mock/MockBlastPoints.sol";
 import {MockBlastWETH} from "../mock/MockBlastWETH.sol";
 import {MockBlastYield} from "../mock/MockBlastYield.sol";
 
-contract BlastYieldOwnableTwoSteps is BlastYield, OwnableTwoSteps {
+contract BlastERC20RebasingYieldOwnableTwoSteps is BlastERC20RebasingYield, OwnableTwoSteps {
     constructor(
         address _blast,
         address _blastPoints,
         address _blastPointsOperator,
         address _owner,
         address _usdb,
         address _weth
-    ) BlastYield(_blast, _blastPoints, _blastPointsOperator, _owner, _usdb, _weth) OwnableTwoSteps(_owner) {}
+    )
+        BlastERC20RebasingYield(_blast, _blastPoints, _blastPointsOperator, _owner, _usdb, _weth)
+        OwnableTwoSteps(_owner)
+    {}
 
     function claim(address wethReceiver, address usdbReceiver) public onlyOwner {
         _claim(wethReceiver, usdbReceiver);
     }
 }
 
-contract BlastYieldOwnableTwoSteps_Test is TestHelpers {
+contract BlastERC20RebasingYieldOwnableTwoStepsOwnableTwoSteps_Test is TestHelpers {
```

Comment:
have you tried reading this test contract's name? just make it BlastERC20RebasingYield_Test

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/bc6d3e8172dbe58c7c8bbb4f91fff5b5de6f0aff